rpc/wrtc_signaling_answerer.go

```diff
@@ -333,7 +355,7 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 			ans.logger.Warnf("answerer swallowing err %v", err)
 			return
 		}
-		ans.logger.Warnf("caller received err %v of type %T", err, err)
+		ans.logger.Warnf("answerer received err %v of type %T", err, err)
```

Comment:
This was a drive by. I think* answerer was the correct noun here.

Reply:
Yep; thanks for the typo fix.